demos/text_detection_demo/cpp/README.md

```diff
@@ -85,13 +103,13 @@ For example, use the following command line command to run the application:
 ./text_detection_demo \
   -i <path_to_image>/sample.jpg \
   -m_td <path_to_model>/text-detection-0004.xml \
-  -m_tr <path_to_model>/text-recognition-0013.xml \
+  -m_tr <path_to_model>/text-recognition-0014.xml \
   -dt ctc \
   -tr_pt_first \
   -tr_o_blb_nm "logits"
 ```
 
-For `text-recognition-resnet-fc` you should use `simple` decoder for `-dt` option. For other models use `ctc` decoder (default decoder).
+For `text-recognition-resnet-fc` and `text-recognition-0015` you should use `simple` decoder for `-dt` option. For other models use `ctc` decoder (default decoder).
 
```

Comment:
```suggestion

> **NOTE**: In case of composite model encoder and decoder are searched automatically: this means that model encoder should have `encoder` part in its name and model decoder should have `decoder` part. In this case to run the demo specify path to the encoder model (`-m_tr` parameter) and decoder model will be searched in the same path but `encoder` would be replaced with `decoder`. E.g.:
> ```
> model-text-recognition-0015:
>   model_encoder.xml
>   model_decoder.xml
```
moved from previous section